src/software/jetson_nano/thunderloop.cpp

```diff
@@ -191,9 +184,10 @@ Thunderloop::~Thunderloop() {}
 
                     // Log milliseconds since last world received if we are timing out
                     LOG(WARNING)
-                        << "Primitive timeout, overriding with StopPrimitive - Milliseconds since last world: "
-                        << static_cast<int>(nanoseconds_elapsed_since_last_primitive) *
-                               MILLISECONDS_PER_NANOSECOND;
+                        << "Primitive timeout, overriding with StopPrimitive\n"
+                        << "Milliseconds since last world: "
+                        << static_cast<int>(nanoseconds_elapsed_since_last_primitive /
+                                            NANOSECONDS_PER_MILLISECOND);
```

Comment:
Seems like a bad merge. The log was updated in #2803 and should remain

Reply:
oh yeah I was confused about this while merging, i'll change it back